src/locales/nl.yaml

```diff
@@ -247,12 +247,12 @@ app:
         max: Max %{max}
         min: Min %{min}
         min_or_0: Min %{min} or 0
-        password_username: Gebruikersnaam past niet
+        password_username: Gebruikersnaam is te lang
```

Comment:
this doesn't seem right; the error triggers when the password equals the username, not when the username is too long

Reply:
"is niet mogelijk" would be fine with me, though the other translations mention the problem directly:
![image](https://user-images.githubusercontent.com/25269274/169152119-f19422d6-8ac8-43e5-907e-f4b20395cf90.png)